src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -70,7 +70,7 @@ MessagePtr generateBannedMessage(bool confirmedBan)
 
 int stripLeadingReplyMention(const QVariantMap &tags, QString &content)
 {
-    if (!getSettings()->stripReplyMention)
+    if (!getSettings()->stripReplyMention || !getSettings()->showReplyContext)
```

Comment:
I think this is more readable
```suggestion
    if (!getSettings()->stripReplyMention && getSettings()->showReplyContext)
```

Reply:
I suppose you mean `!(getSettings()->stripReplyMention && getSettings()->showReplyContext)`.